server/src/main/java/com/ibm/ws/lars/rest/AssetQueryParameters.java

```diff
@@ -99,12 +100,14 @@ public static AssetQueryParameters create(UriInfo uriInfo) {
      *
      * @return a map from parameter name to condition
```

Comment:
Javadoc still says that a map is returned.

Also the doc needs updated to explain how if the same field name appears multiple times, only the last occurrence will be returned. Previously this was sort of stated by the fact that it returned a map.